content/applications/hr/attendances.rst

```diff
@@ -353,8 +375,8 @@ the :guilabel:`Gantt` button, located next to the :guilabel:`List` button.
 
 The default view presents the current day's information. To present the information for the
 :guilabel:`Week`, :guilabel:`Month`, or :guilabel:`Year`, click on the :guilabel:`Day` button to
-reveal a drop-down, displaying those other options. Select the desired view, and the dashboard will
-update, presenting the selected information. To change the :guilabel:`Day`, :guilabel:`Week`,
+reveal a drop-down, displaying those other options. Select the desired view, and the dashboard
+updates, presenting the selected information. To change the :guilabel:`Day`, :guilabel:`Week`,
 :guilabel:`Month`, or :guilabel:`Year` presented, click the left or right :guilabel:`arrow` buttons
```

Comment:
```suggestion
:guilabel:`Month`, or :guilabel:`Year` presented, click the left or right :guilabel:`arrow` buttons
```
Would arrow emojis work here?

Reply:
I changed it to use an icon instead of an emoji, because the emojis are in a bright blue box, which I don't personally like since it's a different look from our UI.